indico/modules/events/management/__init__.py

```diff
@@ -17,7 +17,8 @@
 
 @signals.menu.sections.connect_via('event-management-sidemenu')
 def _sidemenu_sections(sender, **kwargs):
-    yield SideMenuSection('organization', _("Organization"), 50, icon='list', active=True)
+    yield SideMenuSection('organization', _("Organization"), 60, icon='list', active=True)
```

Comment:
This change affects the order of other items as well (protection has 50 as well so with making organization 60 it now shows up before that item).

Keep 50 for this one and use e.g. 45 for workflows (or 40 and renumber the others below ccordingly)

Reply:
I have set `60` for `organization`, `55` for `workflows` and `70` for `protection`. If I don't change the `protection` here, I will have to change the `room booking` one.